tests/test_bot.py

```diff
@@ -3119,7 +3153,7 @@ async def test_replace_callback_data_copy_message(self, cdc_bot, chat_id):
             bot.callback_data_cache.clear_callback_data()
             bot.callback_data_cache.clear_callback_queries()
 
-    async def test_get_chat_arbitrary_callback_data(self, super_group_id, cdc_bot):
+    async def test_get_chat_arbitrary_callback_data(self, channel_id, cdc_bot):
```

Comment:
why?

Reply:
it's to avoid race conditions with `test_pin_and_unpin_message`, we can just use another chat for pinning